source/Server/Integration/JiraRestClient.cs

```diff
@@ -107,8 +110,10 @@ public async Task<IResultFromExtension<JiraIssue[]>> GetIssues(string[] workItem
                     return ResultFromExtension<JiraIssue[]>.Success(result.Issues);
                 }
 
+                var errorResult = await GetResult<JiraErrorResult>(response);
+
                 errorMessage =
-                    $"Failed to retrieve Jira issues '{string.Join(", ", workItemIds)}' from {baseUrl}. Response Code: {response.StatusCode}{(!string.IsNullOrEmpty(response.ReasonPhrase) ? $" (Reason: {response.ReasonPhrase})" : "")}";
+                    $"Failed to retrieve Jira issues from {baseUrl}. Response Code: {response.StatusCode}{(errorResult.ErrorMessages.Any() ? $" (Errors: {string.Join(", ", errorResult.ErrorMessages)})" : "")}";
```

Comment:
Is a `null` check required for `errorResult`? Is it possible for `GetResult` to return a `null` value?

Reply:
yes it is, good catch. I've updated to enable nullable reference type checking in this file and fixed a couple of checks